### DISCUSSION TITLE
Number fields right allignement

### DISCUSSION THREAD

**Discussion by cianopman** [docusealco/docuseal#229] [Ideas]
Hi, is it possible to right align the number fields by default or by settings. Is it when you are buiilding a sales order with a classic sales order detail's table (ie: SKU, descriprion, qty, unit price for each row and having subtotal and grandtotal).
Thank you

**Comment by omohokcoj** [comment]
@cianopman number field right align option has been adde in the 1.4.8 release
![image](https://github.com/docusealco/docuseal/assets/5418788/1c3f7a32-1889-4fab-b028-31b82c771c7e)

**Reply by cianopman** [reply]
Great
